server/controllers/finance/sale.js

```diff
@@ -1,5 +1,5 @@
 // DEPRECIATED - TO BE REMOVED 
```

Comment:
You could actually just delete this file - it is scheduled to be removed (see the comment in the file), and we never use it.